src/engine/platform/abstract.cpp

```diff
@@ -97,6 +97,14 @@ bool DivDispatch::getWantPreNote() {
   return false;
 }
 
+int DivDispatch::getClockRangeMin() {
```

Comment:
The main problem with having a function here is that we won't have an easy way to access these values without a DivDispatch (e.g. the initial system configurator in Settings).
I got to think of a possible approach...

Reply:
This function is prepared for per-chip clock limit (ex: MSM6295 max possible clock rate - 5MHz according to datasheet; Minimum is 1 MHz, but MSM6295 on some hardware runs at lower than 1 MHz)
Global clock limit is still defined at src/engine/chipUtils.h.